core/imageboard/image.php

```diff
@@ -221,37 +221,22 @@ public static function count_tag(string $tag): int
     public static function count_images(array $tags=[]): int
     {
         global $cache, $database;
-        $tag_count = count($tags);
```

Comment:
These can reduce a ton of database load - I wonder if we can keep them, and skip when needed (eg, have some way for extensions to flag "I will alter the default search results") - or have them hidden behind the `SPEED_HAX` flag since that's defined as trading correctness for performance?

Reply:
I agree, it's not optimal for performance, and since these extensions filter differently per-user, setting up new caching for it doesn't really make much sense. I like the speed hax idea, I can implement that.